sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -738,7 +743,8 @@ export class BlobServiceClient extends StorageClient {
    *   }
    * ```
    *
-   * @example
+   * Example usage with `byPage()`:
```

Comment:
Now that we have a line like this talking about "Example" and "byPage()", we can remove the line that appears a few lines below `// Example for .byPage()`

Same goes for `// Generator syntax .next()` a few lines above